jetty-maven-plugin/pom.xml

```diff
@@ -13,6 +13,7 @@
     <mavenVersion>3.5.0</mavenVersion>
     <pluginToolsVersion>3.5</pluginToolsVersion>
     <bundle-symbolic-name>${project.groupId}.maven.plugin</bundle-symbolic-name>
+    <surefireVersion>2.20.1</surefireVersion>
     <it.debug>false</it.debug>
     <jetty.stopKey>FOOBEER</jetty.stopKey>
```

Comment:
Surely this should be FREEBEER?

Reply:
OMG sorry for the typo I will fix that for sure!!